templates/workloads/services/backend/cf.yml

```diff
@@ -115,4 +115,11 @@ Resources:
 {{include "service-base-properties" . | indent 6}}
       ServiceRegistries: !If [ExposePort, [{RegistryArn: !GetAtt DiscoveryService.Arn, Port: !Ref ContainerPort}], !Ref "AWS::NoValue"]
 
-{{include "addons" . | indent 2}}
\ No newline at end of file
+{{include "addons" . | indent 2}}
+
+Outputs:
+  DiscoveryServiceArn:
```

Comment:
nit: Would you mind renaming the output to 
```suggestion
  DiscoveryServiceARN:
```
so that it's consistent with the environment template's outputs:
https://github.com/aws/copilot-cli/blob/a2c80d4da323218c0a688dfa42ae2efdd4e62486/templates/environment/versions/cf-v1.1.0.yml#L279-L287

Reply:
Thanks for the review.  I have made the changes.